### ISSUE TITLE
Webinar: What to look for in an accessibility audit

### ISSUE THREAD

**Issue by ljgoslin** [a11yproject/a11yproject.com#862]
## Title
>What to look for in an accessibility audit.

### Description
>Unfortunately, the majority of accessibility audits fail to see follow-through by the recipients. If your organization is new to accessibility, it is important to note that there are many types of accessibility audits. Join us for this live webinar to learn the key characteristics and processes that should be involved in a successful audit.

### URL
>https://accessibility.deque.com/what-to-look-for-in-an-accessibility-audit.

### Date
>2019-12-05.

### Type
>Webinar

**Comment by ericwbailey** [comment]
Closed via https://github.com/a11yproject/a11yproject.com/commit/62911d77dc678277006bb5658f58f86c897c6bbe.
